--- 00-02.txt	2019-02-06 18:51:04.000000000 -0500
+++ 00-02.fixed	2019-02-06 18:51:04.000000000 -0500
@@ -54,7 +54,6 @@
 Pacific First Financial Corp. said shareholders approved its acquisition by Royal Trustco Ltd. of Toronto for $ 27 a share , or $ 212 million .
 The thrift holding company said it expects to obtain regulatory approval and complete the transaction by year-end .
 McDermott International Inc. said its Babcock & Wilcox unit completed the sale of its Bailey Controls Operations to Finmeccanica S.p .
-A. for $ 295 million .
 Finmeccanica is an Italian state-owned holding company with interests in the mechanical engineering industry .
 Bailey Controls , based in Wickliffe , Ohio , makes computerized industrial controls systems .
 It employs 2,700 people and has annual revenue of about $ 370 million .
@@ -606,25 +605,23 @@
 '' Mr. Coleman said this week that he would devote the remainder of the political season to positive campaigning , but the truce lasted only hours .
 By Tuesday night , television stations were carrying new ads featuring Mr. Coleman himself raising questions about Mr. Wilder 's sensitivity to rape victims .
 New Jersey : The attacks began when Democratic Rep. James Florio aired an ad featuring a drawing of Pinocchio and a photograph of Mr. Florio 's rival , Republican Rep. Jim Courter .
-`` Remember Pinocchio ?
-'' says a female voice .
-`` Consider Jim Courter .
-'' And then this commercial , produced by Bob Squier , gets down to its own mean and dirty business .
-Pictures of rusted oil drums swim into focus , and the female voice purrs , `` That hazardous waste on his -LCB- Mr. Courter 's -RCB- property -- the neighbors are suing for consumer fraud .
-'' And the nose on Mr. Courter 's face grows .
+`` Remember Pinocchio ?'' says a female voice .
+`` Consider Jim Courter . ''
+And then this commercial , produced by Bob Squier , gets down to its own mean and dirty business .
+Pictures of rusted oil drums swim into focus , and the female voice purrs , `` That hazardous waste on his -LCB- Mr. Courter 's -RCB- property -- the neighbors are suing for consumer fraud . ''
+And the nose on Mr. Courter 's face grows .
 The only fraud involved , cry Mr. Courter 's partisans , is the Florio commercial itself , and so the Courter campaign has responded with its own Pinocchio commercial , produced by Mr. Ailes .
 In this one , the screen fills with photographs of both candidates .
-`` Who 's really lying ?
-'' asks a female voice .
-`` Florio 's lying , '' the voice goes on , because `` the barrel on Courter 's land ... contained heating oil , was cleaned up and caused no pollution .
-'' Mr. Courter 's long nose shrinks while Mr. Florio 's grows .
+`` Who 's really lying ? '' asks a female voice .
+`` Florio 's lying , '' the voice goes on , because `` the barrel on Courter 's land ... contained heating oil , was cleaned up and caused no pollution . ''
+Mr. Courter 's long nose shrinks while Mr. Florio 's grows .
 Who 's telling the truth ?
 Stephen Salmore , a political scientist at New Jersey 's Eagleton Institute , says it 's another example of an ad that 's true but not fully accurate .
 Barrels were dumped on the Courter property , a complaint was made , but there is no evidence the barrels were a serious threat to the environment .
 Even so , according to Mr. Salmore , the ad was `` devastating '' because it raised questions about Mr. Courter 's credibility .
 But it 's building on a long tradition .
-In 1966 , on route to a re-election rout of Democrat Frank O'Connor , GOP Gov. Nelson Rockefeller of New York appeared in person saying , `` If you want to keep the crime rates high , O'Connor is your man .
-''A seat on the Chicago Board of Trade was sold for $ 350,000 , down $ 16,000 from the previous sale last Friday .
+In 1966 , on route to a re-election rout of Democrat Frank O'Connor , GOP Gov. Nelson Rockefeller of New York appeared in person saying , `` If you want to keep the crime rates high , O'Connor is your man . ''
+A seat on the Chicago Board of Trade was sold for $ 350,000 , down $ 16,000 from the previous sale last Friday .
 Seats currently are quoted at $ 331,000 , bid , and $ 350,000 , asked .
 The record price for a full membership on the exchange is $ 550,000 , set *-1 Aug. 31 , 1987 .
 Japanese investment in Southeast Asia is propelling the region toward economic integration .
@@ -658,14 +655,14 @@
 In addition , the U.S. this year offered its own plan for cooperation around the Pacific rim in a major speech by Secretary of State James Baker , following up a proposal made in January by Australian Prime Minister Bob Hawke .
 The Baker proposal reasserts Washington 's intention to continue playing a leading political role in the region .
 `` In Asia , as in Europe , a new order is taking shape , '' Mr. Baker said .
-`` The U.S. , with its regional friends , must play a crucial role in designing its architecture .
-'' But maintaining U.S. influence will be difficult in the face of Japanese dominance in the region .
+`` The U.S. , with its regional friends , must play a crucial role in designing its architecture . ''
+But maintaining U.S. influence will be difficult in the face of Japanese dominance in the region .
 Japan not only outstrips the U.S. in investment flows but also outranks it in trade with most Southeast Asian countries -LRB- although the U.S. remains the leading trade partner for all of Asia -RRB- .
 Moreover , the Japanese government , now the world 's largest aid donor , is pumping far more assistance into the region than the U.S. is .
 While U.S. officials voice optimism about Japan 's enlarged role in Asia , they also convey an undertone of caution .
 `` There 's an understanding on the part of the U.S. that Japan has to expand its functions '' in Asia , says J. Michael Farren , undersecretary of commerce for trade .
-`` If they approach it with a benevolent , altruistic attitude , there will be a net gain for everyone .
-'' Some Asian nations are apprehensive about Washington 's demand that Tokyo step up its military spending to ease the U.S. security burden in the region .
+`` If they approach it with a benevolent , altruistic attitude , there will be a net gain for everyone . ''
+Some Asian nations are apprehensive about Washington 's demand that Tokyo step up its military spending to ease the U.S. security burden in the region .
 The issue is further complicated by uncertainty over the future of the U.S. 's leases on military bases in the Philippines and by a possible U.S. troop reduction in South Korea .
 Many Asians regard a U.S. presence as a desirable counterweight to Japanese influence .
 `` No one wants the U.S. to pick up its marbles and go home , '' Mr. Hormats says .
@@ -784,8 +781,7 @@
 `` She said something like ` You just want to make it easy for the school .
 ' I was dumbfounded , '' Mrs. Ward recalls .
 `` It was like someone had turned a knife in me .
-'' To the astonishment and dismay of her superiors and legal authorities -- and perhaps as a measure of the unpopularity of standardized tests -- Mrs .
-Yeargin won widespread local support .
+'' To the astonishment and dismay of her superiors and legal authorities -- and perhaps as a measure of the unpopularity of standardized tests -- Mrs. Yeargin won widespread local support .
 The school-board hearing at which she was dismissed was crowded with students , teachers and parents who came to testify on her behalf .
 Supportive callers decried unfair testing , not Mrs. Yeargin , on a local radio talk show on which she appeared .
 The show did n't give the particulars of Mrs. Yeargin 's offense , saying only that she helped students do better on the test .
@@ -839,8 +835,8 @@
 Scoring High matched on 64.5 .
 In CAT sections where students ' knowledge of two-letter consonant sounds is tested , the authors noted that Scoring High concentrated on the same sounds that the test does -- to the exclusion of other sounds that fifth graders should know .
 Learning Materials for the fifth-grade contains at least a dozen examples of exact matches or close parallels to test items .
-Rick Brownell , senior editor of Scoring High , says that Messrs. Kaminski and Mehrens are ignoring `` the need students have for becoming familiar with tests and testing format .
-'' He said authors of Scoring High `` scrupulously avoid '' replicating exact questions , but he does n't deny that some items are similar .
+Rick Brownell , senior editor of Scoring High , says that Messrs. Kaminski and Mehrens are ignoring `` the need students have for becoming familiar with tests and testing format . ''
+He said authors of Scoring High `` scrupulously avoid '' replicating exact questions , but he does n't deny that some items are similar .
 When Scoring High first came out in 1979 , it was a publication of Random House .
 McGraw-Hill was outraged .
 In a 1985 advisory to educators , McGraw-Hill said Scoring High should n't be used because it represented a `` parallel form '' of the CAT and CTBS tests .
@@ -1057,13 +1053,11 @@
 Related to that decision , the company said it was converting its Santa Clara , Calif. , factory to a research and development facility .
 A spokesman declined to speculate about possible reductions in force .
 `` This is a company that has invested in capacity additions more aggressively than any other company in the industry and now the industry is growing more slowly and they are suddenly poorly positioned , '' said Michael Stark , chip analyst at Robertson , Stephens & Co .
-`` I think the stock is dead money for a while .
-'' Yesterday 's announcement was made after markets closed .
+`` I think the stock is dead money for a while . ''
+Yesterday 's announcement was made after markets closed .
 U.S. chip makers are facing continued slack demand following a traditionally slow summer .
 Part of the problem is that chip buyers are keeping inventories low because of jitters about the course of the U.S. economy .
-INGERSOLL-RAND Co .
--LRB- Woodcliff Lake , N.J .
--RRB- -- William G. Kuhns , former chairman and chief executive officer of General Public Utilities Corp. , was elected a director of this maker of industrial and construction equipment , increasing board membership to 10 .
+INGERSOLL-RAND Co . -LRB- Woodcliff Lake , N.J . -RRB- -- William G. Kuhns , former chairman and chief executive officer of General Public Utilities Corp. , was elected a director of this maker of industrial and construction equipment , increasing board membership to 10 .
 The dollar posted gains against all major currencies yesterday , buoyed by persistent Japanese demand for U.S. bond issues .
 While market sentiment remains cautiously bearish on the dollar based on sluggish U.S. economic indicators , dealers note that Japanese demand has helped underpin the dollar against the yen and has kept the U.S. currency from plunging below key levels against the mark .
 At the same time , dealers said the U.S. unit has been locked into a relatively narrow range in recent weeks , in part because the hefty Japanese demand for dollars has been offset by the mark 's strength , resulting in a stalemate .
@@ -1186,9 +1180,8 @@
 They say that many vehicles classed as commercial light trucks actually carry more people than cargo and therefore should have the same safety features as cars .
 They did n't have much luck during the Reagan administration .
 `` But now , there seems to be a fairly systematic effort to address the problem , '' said Chuck Hurley , vice president of communications for the Insurance Institute for Highway Safety .
-`` We 're in a very different regulatory environment .
-'' Sen. John Danforth -LRB- R. , Mo .
--RRB- praised the department 's actions , noting that rollover crashes account for almost half of all light-truck deaths .
+`` We 're in a very different regulatory environment . ''
+Sen. John Danforth -LRB- R. , Mo . -RRB- praised the department 's actions , noting that rollover crashes account for almost half of all light-truck deaths .
 `` We could prevent many of these fatalities with minimum roof-crush standards , '' he said .
 Sen. Danforth and others also want the department to require additional safety equipment in light trucks and minivans , including air bags or automatic seat belts in front seats and improved side-crash protection .
 The department 's roof-crush proposal would apply to vehicles weighing 10,000 pounds or less .
@@ -1259,19 +1252,18 @@
 `` What 's different is that it is happening with young wines just coming out .
 We 're seeing it partly because older vintages are growing more scarce .
 '' Wine auctions have almost exhausted the limited supply of those wines , Mr. Shapiro continued : `` We 've seen a dramatic decrease in demand for wines from the '40s and '50s , which go for $ 300 to $ 400 a bottle .
-Some of the newer wines , even at $ 90 to $ 100 a bottle or so , almost offer a bargain .
-'' Take Lake Vineyard Cabernet from Diamond Creek .
+Some of the newer wines , even at $ 90 to $ 100 a bottle or so , almost offer a bargain . ''
+Take Lake Vineyard Cabernet from Diamond Creek .
 It 's made only in years when the grapes ripen perfectly -LRB- the last was 1979 -RRB- and comes from a single acre of grapes that yielded a mere 75 cases in 1987 .
-Owner Al Brownstein originally planned to sell it for $ 60 a bottle , but when a retailer in Southern California asked , `` Is that wholesale or retail ?
-'' he re-thought the matter .
+Owner Al Brownstein originally planned to sell it for $ 60 a bottle , but when a retailer in Southern California asked , `` Is that wholesale or retail ? '' he re-thought the matter .
 Offering the wine at roughly $ 65 a bottle wholesale -LRB- $ 100 retail -RRB- , he sent merchants around the country a form asking them to check one of three answers : 1 -RRB- no , the wine is too high -LRB- 2 responses -RRB- ; 2 -RRB- yes , it 's high but I 'll take it -LRB- 2 responses -RRB- ; 3 -RRB- I 'll take all I can get -LRB- 58 responses -RRB- .
 The wine was shipped in six-bottle cases instead of the usual 12 , but even at that it was spread thin , going to 62 retailers in 28 states .
 `` We thought it was awfully expensive , '' said Sterling Pratt , wine director at Schaefer 's in Skokie , Ill. , one of the top stores in suburban Chicago , `` but there are people out there with very different opinions of value .
 We got our two six-packs -- and they 're gone .
 '' Mr. Pratt remarked that he thinks steeper prices have come about because producers do n't like to see a hit wine dramatically increase in price later on .
 Even if there is consumer resistance at first , a wine that wins high ratings from the critics will eventually move .
-`` There may be sticker-shock reaction initially , '' said Mr. Pratt , `` but as the wine is talked about and starts to sell , they eventually get excited and decide it 's worth the astronomical price to add it to their collection .
-'' `` It 's just sort of a one-upsmanship thing with some people , '' added Larry Shapiro .
+`` There may be sticker-shock reaction initially , '' said Mr. Pratt , `` but as the wine is talked about and starts to sell , they eventually get excited and decide it 's worth the astronomical price to add it to their collection . ''
+`` It 's just sort of a one-upsmanship thing with some people , '' added Larry Shapiro .
 `` They like to talk about *-2 having the new Red Rock Terrace -LCB- one of Diamond Creek 's Cabernets -RCB- or the Dunn 1985 Cabernet , or the Petrus .
 Producers have seen this market opening up and they 're now creating wines that appeal to these people .
 '' That explains why the number of these wines is expanding so rapidly .
@@ -1404,9 +1396,7 @@
 `` We have a long history of maintaining an open direct-investment policy , '' Mr. Dallara says .
 `` U.S. investors should have a greater opportunity at direct investment '' in Japan .
 The Japanese fret openly about the U.S. public 's rancor .
-One clear sign of Japan 's nervousness came this week , when a spokesman for Japan 's Foreign Ministry devoted nearly all of a regular , half-hour briefing for foreign journalists to the subject of recent Japanese investments in the U.S. .
-`` We believe that it is vitally important for those Japanese business interests -LCB- in the U.S. .
--RCB- to be more aware of the emotions and concerns of the American people , '' said the spokesman , Taizo Watanabe .
+`` We believe that it is vitally important for those Japanese business interests -LCB- in the U.S. . -RCB- to be more aware of the emotions and concerns of the American people , '' said the spokesman , Taizo Watanabe .
 At the same time , though , he chastised the media for paying such close attention to Japanese investment when other foreign countries , notably Britain , are acquiring more American assets .
 Fears that Japanese investors are buying up America have escalated sharply in the past several weeks , with Sony Corp. 's purchase of Columbia Pictures Entertainment Inc. from Coca-Cola Co. and Mitsubishi Estate Co. 's acquisition of a 51 % holding in Rockefeller Group , the owner of some of midtown Manhattan 's most exclusive real estate .
 Even before those moves added fuel , the fires of discontent had been well stoked by the highly publicized experience in Japan of one U.S. investor , T. Boone Pickens Jr .
@@ -1536,22 +1526,17 @@
 Some banks are already moving in that direction , according to Alvin T. Sale , marketing director at First Union Corp. in Charlotte .
 First Union , he says , now has packages for seven customer groups .
 Soon , it will split those into 30 .
-Says Mr. Sale : `` I think more banks are starting to realize that we have to be more like the department store , not the boutique .
-'' IRAs .
+Says Mr. Sale : `` I think more banks are starting to realize that we have to be more like the department store , not the boutique . '' IRAs .
 SHAREDATA Inc. said it will amend a registration statement filed with the Securities and Exchange Commission to delete a plan to sell 500,000 newly issued common shares .
 The Chandler , Ariz. , company said it will resubmit the registration to cover only the 2.3 million warrants , each exercisable for the purchase of one common share .
 Currently , ShareData has about 4.1 million common shares outstanding .
 ShareData develops and markets low-cost software , peripheral equipment and accessories for computers .
-Five things you can do for $ 15,000 or less : 1 .
-Buy a new Chevrolet .
-2 .
-Take a Hawaiian vacation .
-3 .
-Send your child to a university .
-4 .
-Buy a diamond necklace .
-5 .
-Make a lasting difference in the regulatory life of an American savings-and-loan association through the Foster Corporate Parents Plan .
+Five things you can do for $ 15,000 or less :
+1. Buy a new Chevrolet .
+2. Take a Hawaiian vacation .
+3. Send your child to a university .
+4. Buy a diamond necklace .
+5. Make a lasting difference in the regulatory life of an American savings-and-loan association through the Foster Corporate Parents Plan .
 Americans today spend $ 15,000 like pocket change -- they do n't think much about it .
 But for an ailing savings-and-loan association -- teetering on insolvency -- it can lead to safety from imminent demise and to a future full of promise .
 Your $ 15,000 will help keep a needy savings and loan solvent -- and out of the federal budget deficit .
@@ -1570,8 +1555,7 @@
 Separately , John Phelan told a closed House subcommittee meeting in Washington that he would support Securities and Exchange Commission halts of program trading during market emergencies .
 But the New York Stock Exchange chairman said he does n't support reinstating a `` collar '' on program trading , arguing that firms could get around such a limit .
 The Chicago Merc said a new one-hour price limit would take effect in its Standard & Poor 's 500 stock-index futures pit once S&P 500 futures fell 20 index points -- the equivalent of about a 150-point drop in the Dow Jones Industrial Average .
-If the 20-point limit is triggered after 1:30 p.m .
-Chicago time , it would remain in effect until the normal close of trading at 3:15 p.m .
+If the 20-point limit is triggered after 1:30 p.m. Chicago time , it would remain in effect until the normal close of trading at 3:15 p.m.
 With the limit in effect , members would be able to execute trades at the limit price or at higher prices , but not below it .
 The exchange said it decided a new circuit breaker was needed following a review of the tumultuous trading in stocks and stock-index futures on Friday Oct. 13 , when the Dow Jones industrials plunged 190 points and stock-index futures prices skidded as well .
 Late that afternoon the S&P 500 stock-index futures contract fell a total of 30 index points , hitting a Merc circuit breaker limit that remained in effect for the rest of the trading session .
@@ -1589,8 +1573,8 @@
 In a post-hearing news conference , Mr. Phelan , who has publicly expressed concern about market volatility , said he told the House finance and telecommunications subcommittee that he would support the program-trading halt proposal `` providing the SEC would be comfortable with the language '' in a bill .
 The program-trading issue is heating up on Capitol Hill as it is on Wall Street , and several legislators want to grant the SEC the power to shut off the programs when trading becomes too volatile .
 SEC Chairman Richard Breeden has said he would be willing to consider circuit breakers that have preset trigger points , but he does n't want discretionary power to stop programs .
-A House aide suggested that Mr. Phelan was so `` vague and mushy '' that it was the kind of meeting where people of all viewpoints could `` come out *-2 feeling good .
-'' At one point , Mr. Phelan angered the subcommittee 's chairman , Rep. Edward Markey -LRB- D. , Mass. -RRB- , by not going much beyond what already had been reported in the morning newspapers .
+A House aide suggested that Mr. Phelan was so `` vague and mushy '' that it was the kind of meeting where people of all viewpoints could `` come out *-2 feeling good . ''
+At one point , Mr. Phelan angered the subcommittee 's chairman , Rep. Edward Markey -LRB- D. , Mass. -RRB- , by not going much beyond what already had been reported in the morning newspapers .
 `` Markey said we could have done this in public '' because so little sensitive information was disclosed , the aide said .
 Mr. Phelan then responded that he would have been happy just writing a report to the panel , the aide added .
 At another point during the hearing , Rep. Markey asked Mr. Phelan what would be discussed at a New York exchange board meeting today .
@@ -1785,8 +1769,7 @@
 Many of them recently have been spending a lot of money on public relations and advertising to improve their images , but they should realize that the most important thing is real change , not changing people 's perceptions .
 If the Japanese companies are seriously considering their survival , they could do at least three things to improve the situation : raise salaries higher than those of financial institutions ; improve working conditions -LRB- better offices and more vacations , for example -RRB- ; accept and hire more labor from outside Japan .
 Hiroshi AsadaIn reference to your Oct. 9 page-one article `` Barbara Bush Earns Even Higher Ratings Than the President , '' it is regrettable that you must continually define blacks by our negatives : `` Among liberals , 60 % have positive views of her , while 50 % approve of the president 's job performance .
-In part , this may reflect the fact that ` she speaks a more progressive language ' than her husband , as Columbia 's Prof .
--LCB- Ethel -RCB- Klein puts it .
+In part , this may reflect the fact that ` she speaks a more progressive language ' than her husband , as Columbia 's Prof. -LCB- Ethel -RCB- Klein puts it .
 Among professionals , 76 % have a favorable opinion of her , compared to 62 % who approve of her husband 's performance .
 While a quarter of black voters disapprove of Mr. Bush 's handling of his job , only 15 % have a negative view of his spouse .
 '' The statistics imply that three-quarters of blacks approve of Mr. Bush 's job performance and 85 % of blacks approve of Mrs. Bush .
@@ -1818,8 +1801,7 @@
 Also , Mr. Canepa received a two-week suspension `` in a principal capacity .
 '' Without admitting or denying wrongdoing , they consented to findings that they had inaccurately represented the firm 's net capital , maintained inaccurate books and records , and made other violations .
 Mr. Canepa confirmed he had consented to the sanctions but declined to comment further .
-Weatherly Securities Corp. , New York , and three of its principals -- Dell Eugene Keehn and William Northy Prater Jr. , both of Mercer Island , Wash. , and Thomas Albert McFall , of Red Bank , N.J .
--- consented to a fine of $ 20,000 .
+Weatherly Securities Corp. , New York , and three of its principals -- Dell Eugene Keehn and William Northy Prater Jr. , both of Mercer Island , Wash. , and Thomas Albert McFall , of Red Bank , N.J. -- consented to a fine of $ 20,000 .
 Without admitting or denying wrongdoing , they consented to findings that they failed to return funds owed to customers in connection with a limited-partnership offering .
 Reached at his office , Mr. McFall , currently chairman , said , `` An implication that we failed to return investor funds is inappropriate and inaccurate .
 '' He described the situation as `` an escrow problem , a timing issue , '' which he said was rapidly rectified , with no losses to customers .
@@ -1892,12 +1874,10 @@
 So both sides accepted the compromise , which would lead to the first lifting of the minimum wage since a four-year law was enacted in 1977 , raising the wage to $ 3.35 an hour from $ 2.65 .
 Under the measure passed yesterday , the minimum wage would rise to $ 3.80 next April .
 The Senate plans to take up the measure quickly and is expected to pass it .
-`` There are no smiles about this bill , '' Rep. Pat Williams -LRB- D. , Mont .
--RRB- said during House floor debate yesterday .
+`` There are no smiles about this bill , '' Rep. Pat Williams -LRB- D. , Mont. -RRB- said during House floor debate yesterday .
 But `` because it 's all we 've got , I 'm going to vote for it .
 '' While the minimum wage had traditionally been pegged at half the average U.S. manufacturing wage , the level of $ 4.25 an hour in 1991 will still be less than 35 % of average factory pay , Mr. Williams said .
-But Rep. Marge Roukema -LRB- R. , N.J .
--RRB- instead praised the House 's acceptance of a new youth `` training '' wage , a subminimum that GOP administrations have sought for many years .
+But Rep. Marge Roukema -LRB- R. , N.J. -RRB- instead praised the House 's acceptance of a new youth `` training '' wage , a subminimum that GOP administrations have sought for many years .
 Adopting a training-wage policy means `` getting beyond the nickel and diming of the minimum wage , '' Mrs. Roukema said .
 Policy makers regard the youth wage as helping to limit the loss of jobs from an increase in the minimum wage , but they have lately touted it as necessary *-2 to help impart job skills to entrants into the work force .
 Labor unions and Democrats long fought the idea , but recently acceded to it in the face of Bush administration insistence .
@@ -1965,8 +1945,7 @@
 The sweeping nature of the bill draws a variety of special interest amendments , running from an import exemption for a California airplane museum to a small but intriguing struggle among sugar producing nations over the fate of Panama 's quota of exports to the profitable U.S. market .
 Panama was stripped of this right because of U.S. differences with the Noriega regime , but the Central American country would have received a quota of 30,537 metric tons over a 21-month period ending Sept. 30 , 1990 .
 About a quarter of this share has already been reallocated , according to the industry , but the remaining 23,403 tons are still a lucrative target for growers because the current U.S. price of 18 cents a pound runs as much as a nickel a pound above the world rate .
-The potential sales are nearly $ 9.3 million , and House Majority Whip William Gray -LRB- D. , Pa .
--RRB- began the bidding this year by proposing language that the quota be allocated to English-speaking countries of the Caribbean , such as Jamaica and Barbados .
+The potential sales are nearly $ 9.3 million , and House Majority Whip William Gray -LRB- D. , Pa. -RRB- began the bidding this year by proposing language that the quota be allocated to English-speaking countries of the Caribbean , such as Jamaica and Barbados .
 Rep. Jerry Lewis , a conservative Californian , added a provision of his own intended to assist Bolivia , and the Senate then broadened the list further by including all countries in the U.S. Caribbean Basin initiate as well as the Philippines - backed by the powerful Hawaii Democrat Sen. Daniel Inouye .
 Jamaica , wary of upsetting its Caribbean Basin allies , has apparently instructed its lobbyist to abandon the provision initially drafted by Mr. Gray , but the greater question is whether Mr. Inouye , who has strong ties to the sugar industry , is able to insert a claim by the Philippines .
 In separate floor action , the House waived budget restrictions and gave quick approval to $ 3.18 billion in supplemental appropriations for law enforcement and anti-drug programs in fiscal 1990 .
@@ -2006,8 +1985,7 @@
 The Soviets , who normally have few clients other than the state , will get `` exposure to a market system , '' he says .
 Americans will learn more about making products for the Soviets .
 Mr. Droz says the Soviets could even help U.S. designers renew their sense of purpose .
-`` In Moscow , they kept asking us things like , ` Why do you make 15 different corkscrews , when all you need is one good one ?
-' '' he says .
+`` In Moscow , they kept asking us things like , ` Why do you make 15 different corkscrews , when all you need is one good one ? ' '' he says .
 `` They got us thinking maybe we should be helping U.S. companies improve existing products rather than always developing new ones .
 '' Seed for Jail Solution Fails to Take Root IT 'S A TWO BIRDS with one stone deal : Eggers Group architects propose using grain elevators to house prisoners .
 It would ease jail overcrowding while preserving historic structures , the company says .
@@ -2085,8 +2063,7 @@
 The broader question is where the Senate stands on the issue .
 While the Senate Commerce Committee has approved legislation similar to the House bill on airline leveraged buy-outs , the measure has n't yet come to the full floor .
 Although the legislation would apply to acquisitions involving any major airline , it is aimed at giving the Transportation Department the chance to review in advance transactions financed by large amounts of debt .
-`` The purpose of the bill is to put the brakes on airline acquisitions that would so load a carrier up with debt that it would impede safety or a carrier 's ability to compete , '' Rep. John Paul Hammerschmidt , -LRB- R. , Ark .
--RRB- said .
+`` The purpose of the bill is to put the brakes on airline acquisitions that would so load a carrier up with debt that it would impede safety or a carrier 's ability to compete , '' Rep. John Paul Hammerschmidt , -LRB- R. , Ark. -RRB- said .
 The bill , as it was approved by the House Public Works and Transportation Committee , would give the Transportation Department up to 50 days * to review any purchase of 15 % or more of the stock in an airline .
 The department would be required to block the buy-out if the acquisition is likely to financially weaken a carrier so that safety would be impaired ; its ability to compete would be sharply diminished ; it would be put into foreign control ; or if the transaction would result in the sale of airline-related assets -- unless selling such assets had an overriding public benefit .
 The House approved an amendment offered by Rep. Peter DeFazio -LRB- D. , Ore. -RRB- that would , in addition to the previous criteria , also require the department to block the acquisition of an airline if the added debt incurred were likely to result in a reduction in the number of the carrier 's employees , or their wages or benefits .
@@ -2207,8 +2184,7 @@
 The base rate on corporate loans at large U.S. money center commercial banks .
 FEDERAL FUNDS : 9 1\/2 % high , 8 3\/4 % low , 8 3\/4 % near closing bid , 9 % offered .
 Reserves traded among commercial banks for overnight use in amounts of $ 1 million or more .
-Source : Fulton Prebon -LRB- U.S.A .
--RRB- Inc .
+Source : Fulton Prebon -LRB- U.S.A. -RRB- Inc .
 DISCOUNT RATE : 7 % .
 The charge on loans to depository institutions by the New York Federal Reserve Bank .
 CALL MONEY : 9 3\/4 % .
@@ -2221,14 +2197,12 @@
 Typical rates in the secondary market : 8.60 % one month ; 8.55 % three months ; 8.35 % six months .
 BANKERS ACCEPTANCES : 8.50 % 30 days ; 8.48 % 60 days ; 8.30 % 90 days ; 8.15 % 120 days ; 8.07 % 150 days ; 7.95 % 180 days .
 Negotiable , bank-backed business credit instruments typically financing an import order .
-LONDON LATE EURODOLLARS : 8 3\/4 % to 8 5\/8 % one month ; 8 13\/16 % to 8 11\/16 % two months ; 8 3\/4 % to 8 5\/8 % three months ; 8 5\/8 % to 8 1\/2 % four months ; 8 1\/2 % to 8 7\/16 % five months ; 8 1\/2 % to 8 3\/8 % six months .
-LONDON INTERBANK OFFERED RATES -LRB- LIBOR -RRB- : 8 3\/4 % one month ; 8 3\/4 % three months ; 8 1\/2 % six months ; 8 7\/16 % one year .
+LONDON LATE EURODOLLARS : 8 3\/4 % to 8 5\/8 % one month ; 8 13\/16 % to 8 11\/16 % two months ; 8 3\/4 % to 8 5\/8 % three months ; 8 5\/8 % to 8 1\/2 % four months ; 8 1\/2 % to 8 7\/16 % five months ; 8 1\/2 % to 8 3\/8 % six months . LONDON INTERBANK OFFERED RATES -LRB- LIBOR -RRB- : 8 3\/4 % one month ; 8 3\/4 % three months ; 8 1\/2 % six months ; 8 7\/16 % one year .
 The average of interbank offered rates for dollar deposits in the London market based on quotations at five major banks .
 FOREIGN PRIME RATES : Canada 13.50 % ; Germany 9 % ; Japan 4.875 % ; Switzerland 8.50 % ; Britain 15 % .
 These rate indications are n't directly comparable ; lending practices vary widely by location .
 TREASURY BILLS : Results of the Monday , October 30 , 1989 , auction of short-term U.S. government bills , sold at a discount from face value in units of $ 10,000 to $ 1 million : 7.78 % 13 weeks ; 7.62 % 26 weeks .
-FEDERAL HOME LOAN MORTGAGE CORP .
--LRB- Freddie Mac -RRB- : Posted yields on 30-year mortgage commitments for delivery within 30 days .
+FEDERAL HOME LOAN MORTGAGE CORP. -LRB- Freddie Mac -RRB- : Posted yields on 30-year mortgage commitments for delivery within 30 days .
 9.82 % , standard conventional fixed-rate mortgages ; 8.25 % , 2 % rate capped one-year adjustable rate mortgages .
 Source : Telerate Systems Inc .
 FEDERAL NATIONAL MORTGAGE ASSOCIATION -LRB- Fannie Mae -RRB- : Posted yields on 30 year mortgage commitments for delivery within 30 days -LRB- priced at par -RRB- 9.75 % , standard conventional fixed-rate mortgages ; 8.70 % , 6\/2 rate capped one-year adjustable rate mortgages .
@@ -2244,7 +2218,7 @@
 '' Mr. Bernstein said he will stay until Dec. 31 and work with his successor , who is to be named soon .
 Mr. Newhouse , meanwhile , insisted that he is n't unhappy with Mr. Bernstein or the performance of Random House , the largest trade publishing house in the U.S. .
 The company said the publisher 's annual sales volume increased to $ 800 million from $ 40 million during Mr. Bernstein 's tenure .
-`` Bob has handled the extraordinary growth of the company quite brilliantly , '' said Mr. Newhouse .
+`` Bob has handled the extraordinary growth of the company quite brilliantly , '' said Mr. Newhouse.
 `` The company is doing well , it 's stable , it 's got good people .
 Bob has an agenda and this seemed like the natural time .
 '' Publishing officials believe that while Random House has enjoyed spectacular growth and has smoothly integrated many acquisitions in recent years , some of the bigger ones have n't been absorbed so easily .
@@ -2435,8 +2409,7 @@
 The council backed the audit commission 's stand that the swap transactions are illegal .
 Although the Hammersmith and Fulham council was by far the most active local authority engaging in such capital-markets transactions , the court decision could set a precedent for similar transactions by 77 other local councils .
 `` While this court ruling was only on Hammersmith , it will obviously be very persuasive in other cases of a similar nature , '' a solicitor representing one of the banks said .
-Already , 10 local councils have refused to honor fees and payments to banks incurred during various swaps dealings .
-Other financial institutions involved include Barclays Bank PLC , Midland Bank PLC , Security Pacific Corp. , Chemical Banking Corp. 's Chemical Bank , Citicorp 's Citibank and Mitsubishi Finance International .
+Already , 10 local councils have refused to honor fees and payments to banks incurred during various swaps dealings . Other financial institutions involved include Barclays Bank PLC , Midland Bank PLC , Security Pacific Corp. , Chemical Banking Corp. 's Chemical Bank , Citicorp 's Citibank and Mitsubishi Finance International .
 If the banks exhaust all avenues of appeal , it is possible that they would seek to have the illegality ruling work both ways , some market sources said .
 Banks could seek to recover payments to local authorities in instances where the banks made net payments to councils .
 Officials from the various banks involved are expected to meet during the next few days to consider other arrangements with local authorities that could be questionable .
@@ -2635,8 +2608,7 @@
 Currently , the government charges nothing for such filings .
 Proponents of the funding arrangement predict that , based on recent filing levels of more than 2,000 a year , the fees will yield at least $ 40 million this fiscal year , or $ 10 million more than the budget cuts .
 `` When you do that , there is not a cut , but there is in fact a program increase of $ 5 million '' each for the FTC and the Justice Department , Rep. Neal Smith -LRB- D. , Iowa -RRB- said during House debate .
-But Rep. Don Edwards -LRB- D. , Calif .
--RRB- responded that a recession could stifle merger activity , reducing the amount of fees collected .
+But Rep. Don Edwards -LRB- D. , Calif. -RRB- responded that a recession could stifle merger activity , reducing the amount of fees collected .
 The antitrust staffs of both the FTC and Justice Department were cut more than 40 % in the Reagan administration , and enforcement of major merger cases fell off drastically during that period .
 `` Today is not the time to signal that Congress in any way sanctions the dismal state into which antitrust enforcement has fallen , '' Mr. Edwards argued .
 Any money in excess of $ 40 million collected from the fees in fiscal 1990 would go to the Treasury at large .
@@ -2656,8 +2628,7 @@
 Price records are being set at auctions this week .
 At Christie 's , a folio of 21 prints from Alfred Stieglitz 's `` Equivalents '' series sold for $ 396,000 , a single-lot record .
 Other works also have been exceeding price estimates .
-In part , prices reflect development of a market structure based on such variables as the number of prints .
-This information used to be poorly documented and largely anecdotal , says Beth Gates-Warren of Sotheby 's .
+In part , prices reflect development of a market structure based on such variables as the number of prints . This information used to be poorly documented and largely anecdotal , says Beth Gates-Warren of Sotheby 's .
 `` There is finally some sort of sense in the market , '' she says .
 Corporations and museums are among the serious buyers , giving greater market stability , says Robert Persky of the Photograph Collector .
 `` When I see prints going into the hands of institutions , I know they are n't going to come back on the market .
@@ -2794,25 +2765,21 @@
 Bonds due in 2005 have a 7 1\/2 % coupon and are priced at par .
 The 7 5\/8 % bonds due 2009 are priced to yield 7.65 % , and 7 5\/8 % bonds due 2029 are priced at 98 1\/2 to yield 7.74 % .
 The underwriters expect a double-A rating from Moody 's .
-Heiwado Co .
--LRB- Japan -RRB- -- $ 100 million of Eurobonds due Nov. 16 , 1993 , with equity-purchase warrants , indicating a 3 7\/8 % coupon at par , via Daiwa Europe Ltd .
+Heiwado Co. -LRB- Japan -RRB- -- $ 100 million of Eurobonds due Nov. 16 , 1993 , with equity-purchase warrants , indicating a 3 7\/8 % coupon at par , via Daiwa Europe Ltd .
 Each $ 5,000 bond carries one warrant , exercisable from Nov. 30 , 1989 , through Nov. 2 , 1993 , to buy shares at an expected premium of 2 1\/2 % to the closing price when terms are fixed Tuesday .
 Fees 2 1\/4 .
 Svenska Intecknings Garanti Aktiebolaget -LRB- Sweden -RRB- -- 20 billion yen of 6 % Eurobonds due Nov. 21 , 1994 , priced at 101 3\/4 to yield 6.03 % less full fees , via Mitsui Finance International .
 Guaranteed by Svenska Handelsbanken .
 Fees 1 7\/8 .
-Takashima & Co .
--LRB- Japan -RRB- -- 50 million Swiss francs of privately placed convertible notes due March 31 , 1994 , with a fixed 0.25 % coupon at par via Yamaichi Bank -LRB- Switzerland -RRB- .
+Takashima & Co. -LRB- Japan -RRB- -- 50 million Swiss francs of privately placed convertible notes due March 31 , 1994 , with a fixed 0.25 % coupon at par via Yamaichi Bank -LRB- Switzerland -RRB- .
 Put option March 31 , 1992 , at a fixed 107 7\/8 to yield 3.43 % .
 Each 50,000 Swiss franc note is convertible from Nov. 30 , 1989 , to March 16 , 1994 at a 5 % premium over the closing share price Monday , when terms are scheduled to be fixed .
 Fees 1 3\/4 .
-Mitsubishi Pencil Co .
--LRB- Japan -RRB- -- 60 million Swiss francs of privately placed convertible notes due Dec. 31 , 1993 , with a fixed 0.25 % coupon at par via Union Bank of Switzerland .
+Mitsubishi Pencil Co. -LRB- Japan -RRB- -- 60 million Swiss francs of privately placed convertible notes due Dec. 31 , 1993 , with a fixed 0.25 % coupon at par via Union Bank of Switzerland .
 Put option on Dec. 31 , 1991 , at a fixed 106 7\/8 to yield 3.42 % .
 Each 50,000 Swiss franc note is convertible from Dec. 5 , 1989 , to Dec. 31 , 1993 , at a 5 % premium over the closing share price Tuesday , when terms are scheduled to be fixed .
 Fees 1 5\/8 .
-Koizumi Sangyo Corp .
--LRB- Japan -RRB- -- 20 million Swiss francs of 6 1\/2 % privately placed notes due Nov. 29 , 1996 , priced at 99 1\/2 via Dai-Ichi Kangyo Bank -LRB- Schweiz -RRB- .
+Koizumi Sangyo Corp. -LRB- Japan -RRB- -- 20 million Swiss francs of 6 1\/2 % privately placed notes due Nov. 29 , 1996 , priced at 99 1\/2 via Dai-Ichi Kangyo Bank -LRB- Schweiz -RRB- .
 Guarantee by Dai-Ichi Kangyo Bank Ltd .
 Fees 1 3\/4 .
 Although his team lost the World Series , San Francisco Giants owner Bob Lurie hopes to have a new home for them .
@@ -2870,15 +2837,13 @@
 Whereas conventional securities financings are structured to be sold quickly , Wall Street 's new penchant for leveraged buy-outs and junk bonds is resulting in long-term lending commitments that stretch out for months or years .
 `` The recent disarray in the junk bond market suggests that brokers may become longer-term creditors than they anticipated and may face long delays '' in getting their money back , says Jeffrey Bowman , a vice president at S&P , which raised a warning flag for the industry in April when it downgraded CS First Boston .
 `` Wall Street is facing a Catch-22 situation , '' says Mr. Mahoney of Moody 's .
-Merchant banking , where firms commit their own money , `` is getting riskier , and there 's less of it to go around .
-'' In addition , he says , the buy-out business is under pressure `` because of the junk bond collapse , '' meaning that returns are likely to decline as the volume of junk-bond financings shrinks .
+Merchant banking , where firms commit their own money , `` is getting riskier , and there 's less of it to go around . '' In addition , he says , the buy-out business is under pressure `` because of the junk bond collapse , '' meaning that returns are likely to decline as the volume of junk-bond financings shrinks .
 In a leveraged buy-out , a small group of investors acquires a company in a transaction financed largely by borrowing , with the expectation that the debt will be paid with funds generated by the acquired company 's operations or sales of its assets .
 In a recent report , Moody 's said it `` expects intense competition to occur through the rest of the century in the securities industry , which , combined with overcapacity , will create poor prospects for profitability .
 '' It said that the `` temptation for managements to ease this profit pressure by taking greater risks is an additional rating factor .
 '' Both Moody 's and S&P cited First Boston 's reliance in recent years on merchant banking , which *T*-1 has been responsible for a significant portion of the closely held firm 's profit .
 The recent cash squeeze at Campeau Corp. , First Boston 's most lucrative client of the decade , is proving costly to First Boston because it arranged more than $ 3 billion of high-yield , high-risk junk financings for Campeau units .
-In addition , a big loan that First Boston made to Ohio Mattress Co .
-was n't repaid on time when its $ 450 million junk financing for a buy-out of the bedding company was withdrawn .
+In addition , a big loan that First Boston made to Ohio Mattress Co. was n't repaid on time when its $ 450 million junk financing for a buy-out of the bedding company was withdrawn .
 `` These two exposures alone represent a very substantial portion of CS First Boston 's equity , '' Moody 's said .
 `` Total merchant banking exposures are in excess of the firm 's equity .
 '' CS First Boston , however , benefits from the backing of its largest shareholder , Credit Suisse , Switzerland 's third largest bank .
@@ -2942,8 +2907,7 @@
 They must figure that justice has to get done by somebody , but know it wo n't be done by Congress .
 Odyssey Partners Limited Partnership , an investment firm , completed the purchase of May Department Stores Co. 's Caldor discount chain for $ 500 million plus the assumption of $ 52 million in debt .
 Caldor , based in Norwalk , Conn. , operates 118 stores in the Northeast ; it reported revenue of $ 1.6 billion last year .
-May Stores , St. Louis , runs such well-known department stores as Lord & Taylor .
-N.V .
+May Stores , St. Louis , runs such well-known department stores as Lord & Taylor. N.V .
 DSM said net income in the third quarter jumped 63 % as the company had substantially lower extraordinary charges to account for a restructuring program .
 The Dutch chemical group said net income gained to 235 million guilders -LRB- $ 113.2 million -RRB- , or 6.70 guilders a share , from 144 million guilders , or 4.10 guilders a share , a year ago .
 The 32 % state-owned DSM had eight million guilders of extraordinary charges in the latest quarter , mainly to reflect one-time losses in connection with the disposal of some operations .
@@ -3413,8 +3377,7 @@
 Freeport-McMoRan Energy Partners will be liquidated and shares of the new company distributed to the partnership 's unitholders .
 Unitholders will receive two additional 55 cents-a-unit distribution payments before the trust is liquidated in early 1990 , the company said .
 It is expected that common shares equal to the number of units outstanding -- about 108 million on Sept. 30 -- will be issued during the first quarter of 1990 .
-Freeport-McMoRan , the parent company , holds roughly 80 % of the units outstanding .
-Nissan Motor Co. , Japan 's second-largest car maker , announced Wednesday that the parent concern 's pretax earnings in the first half ended last Sept. 30 rose 14 % to 88.32 billion yen -LRB- $ 618.1 million -RRB- from 77.6 billion yen a year earlier .
+Freeport-McMoRan , the parent company , holds roughly 80 % of the units outstanding . Nissan Motor Co. , Japan 's second-largest car maker , announced Wednesday that the parent concern 's pretax earnings in the first half ended last Sept. 30 rose 14 % to 88.32 billion yen -LRB- $ 618.1 million -RRB- from 77.6 billion yen a year earlier .
 Nissan cited strong domestic sales against the backdrop of continuous economic expansion .
 Profit surged 42 % to 40.21 billion yen , or 16.09 yen a share , from 28.36 billion yen , or 11.72 yen a share .
 Sales totaled 1.916 trillion yen , climbing 17 % from 1.637 trillion yen in the year-earlier period .
@@ -3610,8 +3573,7 @@
 The thrift surely could be sold for more than the value of its equity , financial industry executives say .
 Meanwhile , the bad bank with the junk bonds -- and some capital -- might be spun off *-1 to Columbia shareholders , including Mr. Spiegel , who might then have a new career , investors say .
 It is n't clear how much a restructuring would help Columbia stockholders .
-But `` the concept is workable .
-You sell the good bank as an ongoing operation and use some of the proceeds to capitalize the bad bank , '' says thrift specialist Lewis Ranieri of Ranieri Associates in New York .
+But `` the concept is workable . You sell the good bank as an ongoing operation and use some of the proceeds to capitalize the bad bank , '' says thrift specialist Lewis Ranieri of Ranieri Associates in New York .
 Mr. Spiegel 's next career move is a subject of speculation on Wall Street .
 Few people think Mr. Spiegel wants to run a bread-and-butter thrift , which current rules would force Columbia to become .
 `` They are n't really a thrift , '' says Jonathan Gray , a Sanford C. Bernstein analyst .
@@ -3621,8 +3583,7 @@
 Columbia Savings & Loan -LRB- NYSE ; Symbol : CSV -RRB- Business : Savings and loan Year ended Dec. 31 , 1988 : Net income : $ 65 million ; or $ 1.49 a share Third quarter , Sept. 30 , 1989 : Net loss : $ 11.57 a share vs. net income : 37 cents a share Average daily trading volume : 83,206 shares Common shares outstanding : 19.6 million Note : All per-share figures are fully diluted .
 Genetics Institute Inc. , Cambridge , Mass. , said it was awarded U.S. patents for Interleukin-3 and bone morphogenetic protein .
 The patent for Interleukin-3 covers materials and methods used to make the human blood cell growth factor via recombinant DNA technology .
-Sandoz Ltd. has licensed certain manufacturing and marketing rights for Interleukin-3 from Genetics Institute and is conducting preclinical studies with it .
-Interleukin-3 may help in treating blood cell deficiencies associated with cancer treatment , bone marrow transplants and other blood-cell disorders , Genetics Institute said .
+Sandoz Ltd. has licensed certain manufacturing and marketing rights for Interleukin-3 from Genetics Institute and is conducting preclinical studies with it . Interleukin-3 may help in treating blood cell deficiencies associated with cancer treatment , bone marrow transplants and other blood-cell disorders , Genetics Institute said .
 The second patent describes bone morphogenetic protein-1 , a substance that can induce formation of new cartilage .
 The patent covers BMP-1 type proteins and pharmaceutical compositions and methods for treating bone or cartilage defects , Genetics Institute said .
 The company added that it has filed patent applications `` on a large number of different BMP proteins '' and the patent on BMP-1 is the first it has received .
@@ -3639,8 +3600,7 @@
 Metallgesellschaft AG said it agreed to acquire 51 % of Lentjes AG from the Ferdinand Lentjes Foundation .
 Terms were n't disclosed .
 Metallgesellschaft , a diversified Frankfurt , West Germany-based metals group , said it is buying the stake in the specialized engineering company to expand its production of environmental supplies for power plants .
-Lentjes ' product mix of specialized boilers and pipes provides a good fit with its own Lurgi G.m.b .
-H. plant engineering unit , the company said .
+Lentjes ' product mix of specialized boilers and pipes provides a good fit with its own Lurgi G.m.b. H. plant engineering unit , the company said .
 The move is part of a strategy to focus on its core metals trading , processing and plant engineering activities while shedding peripheral units , the company said .
 Lentjes had 1988 sales of 800 million marks -LRB- $ 434.4 million -RRB- and has a current order backlog of 2.5 billion marks .
 The sale comes in place of a planned initial public offering of Lentjes stock .
@@ -3669,8 +3629,7 @@
 The company earlier this year adopted a shareholder-rights plan to ward off unwanted suitors .
 The spokesman said it is the first early retirement plan offered under its two-year cost-control strategy .
 Earlier staff-reduction moves have trimmed about 300 jobs , the spokesman said .
-INTER-TEL Inc .
--LRB- Chandler , Ariz. -RRB- -- Jerry Chapman , managing director of WayMar Associates , was elected a director of this business telecommunications software and systems concern .
+INTER-TEL Inc. -LRB- Chandler , Ariz. -RRB- -- Jerry Chapman , managing director of WayMar Associates , was elected a director of this business telecommunications software and systems concern .
 He increases the board to seven .
 `` Feeding Frenzy '' -LRB- Henry Holt , 326 pages , $ 19.95 -RRB- , a highly detailed account of the Wedtech scandal , begins on a reassuring note .
 Right up front in the preface , co-author William Sternberg gives us an example of his own integrity .
@@ -3685,8 +3644,7 @@
 Instead of 50\/50 it became , on paper only , two-thirds Mariotta , one-third Neuberger , and they were in the program and off to the races .
 Besides being a `` minority-owned company '' Wedtech was located in the South Bronx , a blighted area , made famous by Jimmy Carter in his 1976 presidential campaign .
 The company plugged itself right into Carter campaign rhetoric about rebuilding the South Bronx and kept using the minority -- South Bronx angle through the Reagan '80s .
-Starting with Congressman Mario Biaggi -LRB- now serving a jail sentence -RRB- , the company began a career of bribing federal , state and local public officials and those close to public officials , right up to and including E. Robert Wallach , close friend and adviser to former Attorney General Ed Meese .
-Wedtech did n't just use old fashioned bribery .
+Starting with Congressman Mario Biaggi -LRB- now serving a jail sentence -RRB- , the company began a career of bribing federal , state and local public officials and those close to public officials , right up to and including E. Robert Wallach , close friend and adviser to former Attorney General Ed Meese. Wedtech did n't just use old fashioned bribery .
 It made ample use of the modern techniques of influence peddling , retaining politically connected `` respectable '' law firms , investment bankers and political consultants , including Reagan confidant Lyn Nofzinger .
 When necessary , it sought and received assistance from organized crime .
 Sometimes the bribed became partners in the company .
@@ -3722,8 +3680,7 @@
 Under an accord signed yesterday , the government and Union Bank of Finland would become major shareholders in the new company , each injecting 100 million Finnish markkaa -LRB- $ 23.5 million -RRB- .
 Oy Waertsilae is to contribute 200 million markkaa , most of it as subordinated debt , and take a minority stake in the new company .
 Customers holding contracts for Waertsilae Marine 's undelivered ships are expected to subscribe most of the remaining 170 million markkaa in share capital , government officials said .
-Waertsilae Marine 's biggest creditor is Miami-based Carnival Cruise Lines Inc .
-Carnival , which has three ships on order from Waertsilae Marine , presented claims for $ 1.5 billion damages in the bankruptcy court this week .
+Waertsilae Marine 's biggest creditor is Miami-based Carnival Cruise Lines Inc. Carnival , which has three ships on order from Waertsilae Marine , presented claims for $ 1.5 billion damages in the bankruptcy court this week .
 Waertsilae Marine 's bankruptcy proceedings began Tuesday in a Helsinki court .
 Its plans to be acquired dashed , Comprehensive Care Corp. said it plans to sell most of its psychiatric and drug abuse facilities in California and some other assets to pay its debt and provide working capital .
 In all , the company hopes to repay $ 45 million in debt through the sales , which will completely discharge its secured debt , the company said .
@@ -4151,12 +4108,10 @@
 Hani Zayadi was appointed president and chief executive officer of this financially troubled department store chain , effective Nov. 15 , succeeding Frank Robertson , who is retiring early .
 Mr. Zayadi was previously president and chief operating officer of Zellers Inc. , a retail chain that is owned by Toronto-based Hudson 's Bay Co. , Canada 's largest department store operator .
 Tuesday , October 31 , 1989 The key U.S. and foreign annual interest rates below are a guide to general levels but do n't always represent actual transactions .
-PRIME RATE : 10 1\/2 % .
 The base rate on corporate loans at large U.S. money center commercial banks .
 FEDERAL FUNDS : 9 % high , 8 13\/16 % low , 8 7\/8 % near closing bid , 8 15\/16 % offered .
 Reserves traded among commercial banks for overnight use in amounts of $ 1 million or more .
-Source : Fulton Prebon -LRB- U.S.A .
--RRB- Inc .
+Source : Fulton Prebon -LRB- U.S.A. -RRB- Inc .
 DISCOUNT RATE : 7 % .
 The charge on loans to depository institutions by the New York Federal Reserve Bank .
 CALL MONEY : 9 3\/4 % to 10 % .
@@ -4175,8 +4130,7 @@
 FOREIGN PRIME RATES : Canada 13.50 % ; Germany 9 % ; Japan 4.875 % ; Switzerland 8.50 % ; Britain 15 % .
 These rate indications are n't directly comparable ; lending practices vary widely by location .
 TREASURY BILLS : Results of the Monday , October 30 , 1989 , auction of short-term U.S. government bills , sold at a discount from face value in units of $ 10,000 to $ 1 million : 7.78 % 13 weeks ; 7.62 % 26 weeks .
-FEDERAL HOME LOAN MORTGAGE CORP .
--LRB- Freddie Mac -RRB- : Posted yields on 30-year mortgage commitments for delivery within 30 days .
+FEDERAL HOME LOAN MORTGAGE CORP. -LRB- Freddie Mac -RRB- : Posted yields on 30-year mortgage commitments for delivery within 30 days .
 9.78 % , standard conventional fixed-rate mortgages ; 7.875 % , 2 % rate capped one-year adjustable rate mortgages .
 Source : Telerate Systems Inc .
 FEDERAL NATIONAL MORTGAGE ASSOCIATION -LRB- Fannie Mae -RRB- : Posted yields on 30 year mortgage commitments for delivery within 30 days -LRB- priced at par -RRB- 9.75 % , standard conventional fixed-rate mortgages ; 8.75 % , 6\/2 rate capped one-year adjustable rate mortgages .
@@ -4307,7 +4261,8 @@
 The market also moved at early afternoon on news that Jaguar shares were being temporarily suspended at 746 pence -LRB- $ 11.80 -RRB- each .
 Secretary of State for Trade and Industry Nicholas Ridley said later in the day that the government would abolish its golden share in Jaguar , the luxury auto maker being stalked by General Motors and Ford Motor .
 The golden share dates from Jaguar 's public offering in 1984 and was designed to protect the company from takeover .
-The golden share was scheduled to expire at the beginning of But although the golden share has been waived , a hostile bidder for Jaguar would still have to alter the British concern 's articles of association which ban shareholdings of more than 15 % .
+The golden share was scheduled to expire at the beginning of 1990 .
+But although the golden share has been waived , a hostile bidder for Jaguar would still have to alter the British concern 's articles of association which ban shareholdings of more than 15 % .
 Jaguar shares closed at 869 pence , up 122 pence , on hefty turnover of 9.7 million shares .
 As the London trading session drew to a close , the market was still listening to the parliamentary debate on the economy , with new Chancellor of the Exchequer John Major expected to clarify his approach to the British economy and currency issues .
 On the Frankfurt Stock Exchange , share prices closed higher in fairly thin trading , as selective buying by foreigners helped propel prices .
